liota/dcc_comms/mqtt_dcc_comms.py

```diff
@@ -74,16 +74,17 @@ def __init__(self, edge_system_name, url, port, identity=None, tls_conf=None, qo
 
         self.client_id = client_id
 
+        if self.client_id is None or self.client_id == "":
+            #  local_uuid generated will be the client ID
+            self.client_id = systemUUID().get_uuid(edge_system_name)
+            log.info("generated local uuid will be the client ID")
+        else:
+            log.info("Client ID is provided by user")
```

Comment:
Are these "info" or "debug" level? Especially if we're not recording what the clientID provided was?

Reply:
Changed them to Debug log as it makes more sense to be in Debug mode.